eng/scripts/Language-Settings.ps1

```diff
@@ -437,26 +437,40 @@ function GetExistingPackageVersions ($PackageName, $GroupId = $null) {
   }
 }
 
-function Validate-javascript-DocMsPackages ($PackageInfo, $DocRepoLocation, $DocValidationImageId) { 
-  $fileLocation = ""
-  if ($PackageInfo.DevVersion -or $PackageInfo.Version -contains "beta") {
-    $fileLocation = (Join-Path $DocRepoLocation 'ci-configs/packages-preview.json')
-    if ($PackageInfo.DevVersion) {
-      $PackageInfo.Version = $PackageInfo.DevVersion
-    }
-  }
-  else {
-    $fileLocation = (Join-Path $DocRepoLocation 'ci-configs/packages-latest.json')
+function Validate-javascript-DocMsPackages ($PackageInfo, $PackageInfos, $DocRepoLocation, $DocValidationImageId) { 
```

Comment:
Is `$PackageInfos` unused currently? A quick search shows me only `$PackageInfo` is used and is passed as a named parameter. Just making sure there's no function signature issues.

Reply:
Nvm I have now discovered https://github.com/Azure/azure-sdk-tools/pull/3169